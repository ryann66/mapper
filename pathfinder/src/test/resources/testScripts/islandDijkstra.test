# Find path to node that is not connected

CreateGraph mg

AddNode mg a
AddNode mg b
AddNode mg c
AddNode mg x
AddNode mg y
AddNode mg z

AddEdge mg a b 1
AddEdge mg b c 1
AddEdge mg c a 1
AddEdge mg x y 5
AddEdge mg y z 5
AddEdge mg z x 5

FindPath mg a z