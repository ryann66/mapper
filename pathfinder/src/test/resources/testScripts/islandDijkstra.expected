# Find path to node that is not connected

created multigraph mg

added node a to mg
added node b to mg
added node c to mg
added node x to mg
added node y to mg
added node z to mg

added edge 1.000 from a to b in mg
added edge 1.000 from b to c in mg
added edge 1.000 from c to a in mg
added edge 5.000 from x to y in mg
added edge 5.000 from y to z in mg
added edge 5.000 from z to x in mg

path from a to z:
no path found